app/config.php

```diff
@@ -149,10 +166,19 @@
     HelpCommand::class => factory(function (ContainerInterface $c) {
         return new HelpCommand(
             $c->get('appName'),
-            $c->get(Output::class),
+            $c->get(OutputInterface::class),
             $c->get(Color::class)
         );
     }),
+
+    //Listeners
+    PrepareSolutionListener::class      => object(),
```

Comment:
Basically means just instantiate the class with no args. It is lazy so wouldnt be created until asked for, and is shared, so subsequent requests get the same instance